source/computer/commands.rst

```diff
@@ -63,54 +62,557 @@ Linux/macOS 下有成百上千个命令，每个命令都有众多选项。这
 
     -   其他
 
+        - `df`_
+        - `du`_
         - `ssh`_
         - `scp`_
         - `rsync`_
+        - `wget`_
+        - `top`_
 
 ----
 
 cat
 ---
 
+``cat`` 命令名来源于 con\ **cat**\ enate（即“拼接”）。该命令可以输出某个文件的
+内容，或将多个文件\ **拼接**\ 起来。
+
+将一个文件的内容打印到标准输出::
+
+    $ cat file
+
+将文件 :file:`file1` 和 :file:`file2` 合并到一个文件 :file:`target_file` 中::
+
+    $ cat file1 file2 > target_file
+
+打印文件内容并显示行号::
+
+    $ cat -n file
+
+diff
+----
+
+``diff`` 命令的命名来自 **diff**\ erence。该命令可以用来逐行比较文件的异同::
+
+    $ diff file1 file2
+
+df
+--
+
+``df`` 命令的命名来自 **d**\ isk **f**\ ree（可使用的磁盘空间）。该命令可以获取硬盘被占用了多少空间，
+目前还剩下多少空间等信息。默认显示单位为 KB::
+
+    $ df
+
+以可读性较高的方式来显示信息，即使用 ``-h`` 或 ``--human-readable`` 选项::
+
+    $ df -h
+
+du
+--
+
+``du`` 命令的名字来自 **d**\ isk **u**\ sage（磁盘使用情况）。该命令可以查看磁盘的使用空间。
+但与 `df`_ 命令不同的是，\ ``du`` 命令用于查看文件和目录磁盘使用的空间。
+
+查看当前目录所占磁盘空间::
+
+    # 默认显示当前目录及其所有子目录所占磁盘空间
+    $ du
+    # 若只显示当前目录所占磁盘空间，不显示其任何子目录
+    $ du --max-depth=0
+    # 若只显示当前目录及其一级子目录所占磁盘空间
+    $ du --max-depth=1
+    # 若只显示当前目录及其一级、二级子目录所占磁盘空间
+    $ du --max-depth=2
```

Comment:
Again, the output of `du` is difficult to understand, so we can only talk about `du -h`.

Reply:
done in e3c02cd.